app/frontend/src/features/messages/Messages.tsx

```diff
@@ -63,7 +63,7 @@ type MessageType = keyof typeof labels;
 
 export default function Messages() {
   const history = useHistory();
-  const { type = "chats" } = useParams<{ type: keyof typeof labels }>();
+  const { type = "chats" } = useParams<{ type: MessageType }>();
```

Comment:
```suggestion
  const { type = "chats" } = useParams<{ type: string }>();
```
Same comment as in community page.